api/identifiers/views.py

```diff
@@ -71,7 +71,7 @@ def get_object(self, *args, **kwargs):
 
     def get_default_queryset(self):
         obj = self.get_object()
-        return obj.identifiers.all()
+        return obj.identifiers.all().exclude(category='ark')
```

Comment:
Driveby: Pretty sure the `.all()` doesn't do anything if you're excluding

Reply:
ooh good to know. Thanks!